meta-lmp-base/recipes-sota/custom-sota-client/custom-sota-client_git.bb

```diff
@@ -0,0 +1,28 @@
+DESCRIPTION = "Custom SOTA Client example based on the aktualizr-lite C++ API"
+SECTION = "base"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+inherit pkgconfig cmake systemd
+
+DEPENDS = "jsoncpp boost aktualizr"
+
+BRANCH = "master"
+SRCREV = "05c4180840ef84c21b0632be4a542580d720491f"
+
+SRC_URI = "\
+    file://systemd.service \
+    git://github.com/foundriesio/aktualizr-lite;branch=${BRANCH} \
```

Comment:
Nit, can you invert the order here? Usually the first entry is the source repository, followed by local files and patches.

Reply:
done in a fixup commit that we can squash

Reply:
Seems it wasn't updated here, maybe missing a push or so?